src/openrct2/interface/Viewport.cpp

```diff
@@ -1,4 +1,4 @@
-﻿/*****************************************************************************
+/*****************************************************************************
```

Comment:
I guess I inherited @ZehMatt's curse

Reply:
This happened ever since we switched to VS2019, I never had this once before that. I installed some extension called "Fix file encoding" and it seems to work 50% of the time, so you might wanna give that a try.